File: Lista1/Exercícios_python/Sala/1viagem.py
tempo = float(input("Digite o tempo gasto na viagem, em horas: "))
vm = float(input("Digite a velocidade média, em km/h: "))

distancia = (tempo*vm)
litros = (distancia/12)

print("A velocidade média é: ", vm, "km/h")
print("O tempo gasto na viagem é: ", tempo, "h")
print("A distância percorrida é: ", distancia, "km")
print("A quantidade de litros usados é: ", litros, "l")

File: Lista1/Exercícios_C/Sala/1viagem.c
#include <stdio.h>

int main ()

{
    float t, vm, d, lu;
    printf("Digite o tempo gasto na viagem, em horas: ");
    scanf("%f", &t);
    printf("Digite a velocidade média, em km/h: ");
    scanf("%f", &vm);

    d = t*vm;
    lu = d/12;

    printf("A velocidade média é: %.2f km/h \n", vm);
    printf("O tempo gasto na viagem é: %.2f h \n", t);
    printf("A distância percorrida é: %.2f km \n", d);
    printf("A quantidade de litros usados é: %.2f l \n", lu);
}

